public/app/features/alerting/unified/components/rules/MultipleDataSourcePicker.tsx

```diff
@@ -1,92 +1,30 @@
-// Libraries
-import React, { PureComponent } from 'react';
+import React, { FC, useState } from 'react';
```

Comment:
We're no longer going to use `FC` as per [this PR](https://github.com/grafana/grafana/pull/64722) – rationale is in the PR description :)